src/test/java/org/rulez/demokracia/pdengine/VoteCastTest.java

```diff
@@ -150,22 +129,31 @@ public void cast_vote_records_the_vote_with_same_user_votesCast_when_same_user_i
 	@tested_behaviour("if there was a cast vote from the same user, the old one is deleted")
 	@Test 
 	public void cast_vote_records_the_vote_with_same_user_votesCast_when_the_list_does_not_contain_same_user() {
-		String ballot = voteManager.obtainBallot(adminInfo.voteId, adminInfo.adminKey);
-		List<RankedChoice> theCastVote = new ArrayList<RankedChoice>();
-		Vote vote = getTheVote();
-		vote.canVote = true;
+		init();
 		vote.canUpdate = true;
 		
-		vote.votesCast.clear();
-		vote.votesCast.add(new CastVote("OtherUser1", theCastVote));
-		vote.votesCast.add(new CastVote("OtherUser2", theCastVote));
-		vote.votesCast.add(new CastVote("OtherUser3", theCastVote));
-		vote.votesCast.add(new CastVote("OtherUser4", theCastVote));
+		addCastVote("OtherUser1", theCastVote);
+		addCastVote("OtherUser2", theCastVote);
+		addCastVote("OtherUser3", theCastVote);
+		addCastVote("OtherUser4", theCastVote);
 		
 		voteManager.castVote(adminInfo.voteId, ballot, theCastVote);
 		
 		System.out.println("ASD: " + vote.votesCast.get(4).proxyId);
 		
 		assertTrue(vote.votesCast.get(4).proxyId.equals("test_user_in_ws_context"));
 	}
+	
+	private void init() {
```

Comment:
not a meaningful name

Reply:
As this init is in beginning of every test, it can be part of the setUp, and you are relieved of the terror of naming things ;)